src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphWorkspaceDAO.scala

```diff
@@ -29,6 +29,10 @@ class GraphWorkspaceDAO extends WorkspaceDAO with GraphDAO {
     loadContext(workspaceName, txn) map { context => loadFromVertex[Workspace](context.workspaceVertex) }
```

Comment:
I believe Doug had it use loadFromContext. It looks this was the result of a mistake when I rebased. I'll take a closer look.